python/PyQt6/core/auto_generated/providers/qgsprovidermetadata.sip.in

```diff
@@ -474,7 +474,7 @@ create new empty GeoPackage or FileGeodatabase databases.
       int height,
       double *geoTransform,
       const QgsCoordinateReferenceSystem &crs,
-      const QStringList &createOptions = QStringList() ) /Factory/;
+      const QStringList &creationOptions = QStringList() ) /Factory/;
```

Comment:
This is an API break, as any python code using named arguments will break after this. We can only rename the arguments for methods introduced in 3.44....

Reply:
Should provide overloaded method and deprecate this one or just keep this as is?

Reply:
I don't think it's possible to do this via an overload. I'd just leave as is and add a QGIS 4.0 TODO